nmdc_schema/migration_recursion.py

```diff
@@ -151,18 +97,6 @@ def main(schema_path, input_path, output_path, salvage_prefix, migrator_name):
 
     logger.info(curie_slots)
 
-    curie_migrator = CurieMigrator()
-    curie_migrator.forced_prefix = salvage_prefix
-
-    # iterate over collections, applying CURIe fixes
-    for tdk, tdv in total_dict.items():
-        logger.info(f"Fixing CURIes in {tdk} if necessary")
-
-        total_dict[tdk] = curie_migrator.fix_curies_recursively_by_key(
-            tdv, set(curie_slots))
-
-    # Instantiate an adapter that the migrator can use to manipulate a "database" represented as a Python dictionary.
-    # Also, specify some callback functions, so we can react when certain events occur.
```

Comment:
I'd prefer that this two-line comment remain in the file, since the code that it describes is still in the file.